internal/clients/consul_cluster.go

```diff
@@ -3,22 +3,22 @@ package clients
 import (
 	"context"
 
-	"github.com/hashicorp/hcp-sdk-go/clients/cloud-consul-service/preview/2020-08-26/client/consul_service"
-	consulmodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-consul-service/preview/2020-08-26/models"
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-consul-service/preview/2021-02-04/client/consul_service"
+	consulmodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-consul-service/preview/2021-02-04/models"
 	sharedmodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-shared/v1/models"
 )
 
 // GetConsulClusterByID gets an Consul cluster by its ID
 func GetConsulClusterByID(ctx context.Context, client *Client, loc *sharedmodels.HashicorpCloudLocationLocation,
-	consulClusterID string) (*consulmodels.HashicorpCloudConsul20200826Cluster, error) {
+	consulClusterID string) (*consulmodels.HashicorpCloudConsul20210204Cluster, error) {
```

Comment:
oh gosh this is annoying to have to change everywhere. 😅  just wrote up an SDK ticket for seeing if we can strip out that version number, so these don't have to be updated every time the service version bumps

Reply:
Thank you! That would be great.